src/pipecat/services/simli.py

```diff
@@ -77,35 +93,33 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
         elif isinstance(frame, TTSAudioRawFrame):
             # Send audio frame to Simli
             try:
-                if self._ready:
-                    oldFrame = AudioFrame.from_ndarray(
-                        np.frombuffer(frame.audio, dtype=np.int16)[None, :],
-                        layout=frame.num_channels,
-                    )
-                    oldFrame.sample_rate = frame.sample_rate
-                    if self._pipecat_resampler is None:
-                        self._pipecat_resampler = AudioResampler(
-                            "s16", oldFrame.layout, oldFrame.sample_rate
-                        )
-
-                    resampledFrame = self._simli_resampler.resample(oldFrame)
-                    for frame in resampledFrame:
-                        await self._simli_client.send(frame.to_ndarray().astype(np.int16).tobytes())
-                    return
-                else:
-                    logger.warning(
-                        "Simli Connection is not Initialized properly, passing audio to next processor"
+                oldFrame = AudioFrame.from_ndarray(
+                    np.frombuffer(frame.audio, dtype=np.int16)[None, :],
+                    layout=frame.num_channels,
+                )
+                oldFrame.sample_rate = frame.sample_rate
+                if self._pipecat_resampler is None:
+                    self._pipecat_resampler = AudioResampler(
+                        "s16", oldFrame.layout, oldFrame.sample_rate
                     )
+                    self._pipecat_resampler_event.set()
+                resampledFrame = self._simli_resampler.resample(oldFrame)
+                for frame in resampledFrame:
```

Comment:
`frame` here could be confusing. Maybe `resampled_audio` would be clearer?

```suggestion
                for resampled_audio in resampledFrame:
```